### DISCUSSION TITLE
[BUG] Morph Error

### DISCUSSION THREAD

**Discussion by YesIHaveVerified** [Prodigy-Hacking/ProdigyMathGameHacking#596] [(B) Bug Discussion]
Issue: I clicked morph fix and morph player and my screen froze and I was unable to do anything.

**Comment by ghost** [comment]
I would tell you to move to an Issue, but this has already been reported twice.

**Reply by YesIHaveVerified** [reply]
Oh, this has?

**Reply by ghost** [reply]
Yes. #563 #594

**Comment by YesIHaveVerified** [comment]
Man, i'm stupid.

**Reply by ghost** [reply]
It's fine, just make sure to check open Issues next time ^^

**Comment by YesIHaveVerified** [comment]
I've looked at them and I still haven't found how to actually fix it. Is this still being fixed or is it supposed to be working?

**Reply by ghost** [reply]
A patch is still being developed. We aren't sure what causes the issue.

**Reply by YesIHaveVerified** [reply]
I see, it appears that this worked back then.

**Comment by YesIHaveVerified** [comment]
Either those, or I can't read.
